sdk/data_kusto/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "azure_data_kusto"
+version = "0.1.0"
+description = "Rust wrappers around Microsoft Azure REST APIs - Azure Data Explorer"
+readme = "README.md"
+license = "MIT"
+edition = "2021"
+repository = "https://github.com/azure/azure-sdk-for-rust"
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/azure_data_kusto"
+keywords = ["sdk", "azure", "kusto", "azure-data-explorer"]
+categories = ["api-bindings"]
+
+[dependencies]
+azure_core = { path = "../core", version = "0.1" }
+azure_identity = { path = "../identity", version = "0.1" }
+async-trait = "0.1"
+bytes = "1"
+futures = "0.3"
+http = "0.2"
+serde = "1"
+serde_derive = "1"
```

Comment:
Can we just bring this in through `serde`: `serde = { version = "1.0", features = ["derive"] }`

Reply:
yes :)